# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1296922449 18000
#      Sat Feb 05 11:14:09 2011 -0500
# Node ID d63007ac067a734a55ce410db9fe1727149717a1
# Parent  eda5eabd5d0b315160db0996741e987aa9337121
legend.m: Add demo for inline keys created by two plot commands.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2011-02-05  Ben Abbott  <bpabbott@mac.com>
+
+	* plot/legend.m: Add demo for inline keys created by two plot commands.
+
 2011-02-05  David Bateman  <dbateman@free.fr>
 
 	Bug #32022 and #32343.
 
 	* plot/legend.m (updatelegend): Don't flip plots when resetting the
 	legend. Use unwind_protect to ensure that the resursive flag is reset
 	in case of an error.
 	(getlegenddata): Remove this function and replace its use with the
diff --git a/scripts/plot/legend.m b/scripts/plot/legend.m
--- a/scripts/plot/legend.m
+++ b/scripts/plot/legend.m
@@ -1066,8 +1066,16 @@ endfunction
 
 %!demo
 %! clf
 %! x = 0:0.1:7;
 %! h = plot (x, sin(x), x, cos(x), x, sin(x.^2/10), x, cos(x.^2/10));
 %! title ("Only the sin() objects have keylabels");
 %! legend (h([1, 3]), {"sin(x)", "sin(x^2/10)"}, "location", "southwest");
 
+%!demo
+%! clf
+%! x = 0:0.1:10;
+%! plot (x, sin(x), ";sin(x);")
+%! hold all
+%! plot (x, cos(x), ";cos(x);")
+%! hold off
+
